assets/translations/zh-hans.yaml

```diff
@@ -4916,170 +5976,295 @@ unit:
-    'ui-index': 'UI的HTML索引。'
-    'ui-pl': 'Private-Label company name.'
-    'ui-issues': '使用一个URL地址来发送新的 "提交问题 "报告，而不是将用户发送到Github问题页面。'
-    'telemetry-opt': '遥测报告opt-in。'
-    'auth-user-info-max-age-seconds': '在进行认证提供者组成员同步之前，用户认证令牌的最大存活时间。'
-    'auth-user-info-resync-cron': '重新同步认证提供者组成员资格的默认cron时间表。'
-    'cluster-template-enforcement': '非管理员将被限制只能通过预先批准的RKE模板启动集群。'
-    'auth-user-session-ttl-minutes': '用户认证会话的自定义TTL（以分钟为单位）。'
-    'auth-token-max-ttl-minutes': '自定义一个授权令牌的最大TTL（以分钟为单位）。'
-    'rke-metadata-config': '配置RKE元数据刷新参数。'
-    'ui-banners': '分类横幅是用来在页眉、页脚或两者中显示一个自定义的固定横幅。'
-    'ui-default-landing': '用户在登录后登陆的默认页面。'
-    'harv-vlan': VLAN网络的默认NIC名称。
-    'harv-backup-target': 自定义备份目标用于存储VM备份。
-    'harv-log-level': 配置Harvester服务器日志级别. 默认为信息。
-    'harv-server-version': Harvester服务器版本。
-    'harv-upgrade-checker-enabled': 指定是否启用Harvester升级检查. 默认为true。
-    'harv-upgrade-checker-url': 默认的Harvester升级检查地址。 仅在<code> upgrade-checker-enabled </code> 等于true时使用。
-    'harv-ui-source': 配置如何加载UI资源。
-    'harv-ui-index': 'UI的HTML索引。'
-    'harv-cluster-registration-url': 用于多集群管理的注册URL。
-    'harv-http-proxy': '用于Harvester访问外部服务的HTTP代理。'
-    'harv-additional-ca': '用于TLS验证的自定义CA根证书。'
+    'cacerts': "验证服务器证书所需的 CA 证书。"
+    'cluster-defaults': '创建新集群时覆盖 RKE 默认值。'
+    'engine-install-url': '默认的 Docker 引擎安装 URL（用于大多数节点驱动）。'
+    'engine-iso-url': '默认操作系统安装 URL（用于 vSphere 驱动）。'
+    'engine-newest-version': '此版本发布时最新支持的 Docker 版本。不在支持范围，但比此版本更新的 Docker 版本，会被标记为未测试。'
+    'engine-supported-range': '支持的 Docker 引擎版本的 Semver 范围。不在此范围内的版本将在 UI 中标记为不支持。'
+    'ingress-ip-domain': '用于自动生成 Ingress 主机名的通配符 DNS 域。<ingress-name>.<namespace-name>.<ip address of ingress controller> 会被添加到该域。'
+    'server-url': '默认 {appName} 安装地址。必须使用 HTTPS。你集群中的所有节点都必须能够访问此地址。'
+    'system-default-registry': '用于所有系统 Docker 镜像的私有仓库。'
+    'ui-index': 'Cluster Manager UI 的 HTML 索引位置。'
+    'ui-dashboard-index': '{appName} UI 的 HTML 索引位置。'
+    'ui-offline-preferred': '控制 UI 资产是由服务器容器在本地提供，还是从 ui-index 和 ui-dashboard-index 设置中定义的远程 URL 提供。`动态` 选项将在 {appName} 的生产版本中使用本地资产。'
+    'ui-pl': '私有标签公司名称。'
+    'ui-issues': "使用 URL 地址发送新的`提交问题`报告，而不是让用户发送到 GitHub Issue 页面。"
+    'telemetry-opt': '遥测报告加入。'
+    'auth-user-info-max-age-seconds': '在同步验证提供程序组成员之前，用户验证 Token 的最长存活时间。'
+    'auth-user-info-resync-cron': '重新同步验证提供程序组成员的默认 cron 调度。'
+    'cluster-template-enforcement': '非管理员只能通过预先批准的 RKE 模板启动集群。'
+    'auth-user-session-ttl-minutes': '用户认证会话的自定义 TTL（单位：分钟）。'
+    'auth-token-max-ttl-minutes': '身份验证 Token 的自定义最大 TTL（单位：分钟）。'
+    'kubeconfig-generate-token': '自动为用户生成 Kubeconfig Token。'
+    'kubeconfig-token-ttl-minutes': 'Kubeconfig Token 的自定义最大 TTL（单位：分钟）。'
+    'rke-metadata-config': '配置 RKE 元数据刷新参数。'
+    'ui-banners': '分类横幅用于在页眉、页脚或两者中显示自定义的固定横幅。'
+    'ui-consent-banner': '横幅用于在登录期间，向用户展示自定义同意横幅。'
+    'ui-default-landing': '用户登录后的默认页面。'
+    'brand': 在'/assets/brand'中定义，用于替代主题的文件夹名称
+    'harv-vlan': VLAN 网络的默认网络接口名称。
```

Comment:
```suggestion
    'harv-vlan': 'VLAN 网络的默认网络接口名称。'
```

Reply:
Double check lines from 6019 to 6056. The formats are somewhat confusing here.
Which one is correct? Or maybe all of them are correct? I would suggest keeping them in one form.
`'a':'descrption'`
`'a':descrption`
`'a':"descrption"`

Reply:
The current ones follow the formats of the original English version, so I think it should be fine~

Reply:
cool!